docs/narr/vhosting.rst

```diff
@@ -80,6 +80,8 @@ In the above configuration, we root a :app:`Pyramid` application at
 
 .. _virtual_root_support:
 
+Similar behaviour gives `route_prefix` `Configurator's` param. It will add prefix to all routes.
```

Comment:
* Do not split indices and labels from their section headings.
* We use American spellings. s/b "behavior".
* Wrap to 79 columns.
* Improve the grammar.

Aside from those issues, I do not understand what concept you are trying to express here.